index.html

```diff
@@ -222,33 +222,39 @@ <h2>About me</h2>
                             </div>
                         </div>
                     </section>
-            <!-- start of contact section  -->
 
-            <!--Contact form section added by Charles Andrew-->
+
+                    <!-- start of contact section --> 
+
+                    <!--Contact form section added by Celyne Birungi-->
                     
-                    <section class="contactFormSec">
-                        <div class="body-area">
-                          <div class="contactFormSec_para">
-                            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-                          </div>
-                          
-                          <form action="https://formspree.io/f/xyyangae" method="post">
-                            <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required>
-                            <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
-                            <textarea name="short-writeup" id="bio" class="text-field" cols="30" rows="5" minlength="1" maxlength="500" placeholder="Write me something..." required></textarea>
-                            <div class="form-button">
-                                  <button type="submit" class="about-me-btn btn-form">Get in touch</button>
+                    <section class="contactFormSec">    
+                        <div class="body-area"> 
+
+                            <div class="contactFormSec_para">   
+                                 <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>   
                             </div>
-                                
-                          </form>
-                        </div>
+                            <form action="https://formspree.io/f/xyyangae" method="post">    
+                                <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required>
+                                <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+                                <textarea name="short-writeup" id="bio" class="text-field" cols="30" rows="5" minlength="1" maxlength="500" placeholder="Write me something..." required></textarea>
```

Comment:
- [ ] Kindly make sure the textarea matches the design by making it slightly reddish gray like it is in the design and also please remove the bottom border, as the original design, doesn't have that.

- Your design
<img width="283" alt="image" src="https://user-images.githubusercontent.com/41163571/229757984-46c64fcf-44ec-4dbc-9034-b8cfbb52d614.png">